antispoof/src/antispoof.c

```diff
@@ -162,9 +159,12 @@ static int antispoof_burst(struct pg_brick *brick, enum pg_side from,
 			continue;
 		}
 	}
-	return pg_brick_side_forward(s, from, pkts, pkts_mask, errp);
+	if(s->edge.link == NULL)
```

Comment:
This is a duplication with the code in nop.c, so even if this work, we should add a function pg_brick_edge_side_forward in brick.h that will allow us to have the same code for each brick which
need to forward on only one side.